modules/core/shared/src/main/scala/lucuma/core/model/SiderealTracking.scala

```diff
@@ -150,7 +150,9 @@ object SiderealTracking extends SiderealTrackingOptics {
 
     // Break out our components
     val (ra, dec)   = baseCoordinates
-    val (dRa, dDec) = properMotion
+    val (dRaʹ, dDec) = properMotion
+    // See: https://app.shortcut.com/lucuma/story/1388/proper-motion-calculation
+    val dRa  =  dRaʹ / cos(dec)
```

Comment:
I wasn't sure but dec can be `[90, -90]` inclusive and this becomes undefined. Should I make the return an `Option`? It will be a bit of a problem to handle it upstream but I guess better than an exception

And we all know somebody's going to try